content/en/docs/instrumentation/net/manual.md

```diff
@@ -277,4 +281,13 @@ using var anotherActivity =
         links: links);
 
 // do some work
-```
\ No newline at end of file
+```
+
+## Next steps
+
+If you're not utilizing [instrumentation libraries]({{< relref "automatic" >}}), it's highly recommended that you do so.
```

Comment:
At a minimum I'd suggest the following edits:

- utilizing --> using
- it's highly recommended that you do so --> we highly recommend that you do

But, come to think of it, I'd rather we consider rewriting the sentence to speak of automatic instrumentation first (and this is related to #1042). For example, something like:

> If you're not using [automatic instrumentation](#), we highly recommend that you do.

If you agree, then the sentence that follows will also need to be reworked.

Reply:
Thanks for the update here.